Runtime/Game/Requests/ClassRequests.cs

```diff
@@ -22,31 +26,33 @@ public LootLockerCommonAsset[] GetAssets()
         }
     }
 
+    public class LootLockerDefaultClassLoadout 
+    {
+        public int variation_id { get; set; }
+        public int instance_id { get; set; }
+        public DateTime mounted_at { get; set; }
+        public LootLockerCommonAsset asset { get; set; }
+        public LootLockerRental rental { get; set; }
+    }
 
-    public class LootLockerListCharacterTypesResponse : LootLockerResponse
+    public class LootLockerListClassTypesResponse : LootLockerResponse
     {
-        public LootLockerCharacter_Types[] character_types { get; set; }
+#if LOOTLOCKER_USE_NEWTONSOFTJSON
```

Comment:
We should probably define our own attribute and ifdef in a single place instead.

Reply:
Is this something we should do now or should we not create a new task for this?

Reply:
Yeah, that makes sense. Can you do that?

Reply:
Done!